modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -50,7 +50,17 @@ run.meta.analysis.pft <- function(pft, iterations, random = TRUE, threshold = 1.
   dir.create(pathname, showWarnings = FALSE, recursive = TRUE)
   
   ## Convert data to format expected by pecan.ma
-  jagged.data <- lapply(trait.data, PEcAn.MA::jagify)
+  jagged.data <- lapply(trait.data, PEcAn.MA::jagify, use_ghs)
```

Comment:
This might help avoid ambiguity if arguments get changed in the future.

```suggestion
  jagged.data <- lapply(trait.data, PEcAn.MA::jagify, use_ghs = use_ghs)
```